mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -985,7 +985,7 @@ def send_email(self, title, body):
             body  (str): HTML body of email. This supports
                          simple HTML like bold and italics
         """
-        DeviceApi().send_email(title, body, basename(self.root_dir))
+        EmailApi().send_email(title, body, self.skill_id)
```

Comment:
Consider extending to add attachment support included in NeonSkill class
https://github.com/NeonGeckoCom/neon-skill-utils/blob/dev/neon_utils/skills/neon_skill.py#L314-L329

Reply:
open PR/issue in ovos-backend-client repo :) it will be easier to do this now, maybe together with the neon backend PR